src/pages/Schedule.tsx

```diff
@@ -1,7 +1,323 @@
-import React from "react";
+import {
+  Box,
+  Button,
+  Container,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Heading,
+  Input,
+  Radio,
+  RadioGroup,
+  Select,
+  Slider,
+  SliderFilledTrack,
+  SliderThumb,
+  SliderTrack,
+  Stack,
+  Textarea,
+  Tooltip,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Link } from "react-router-dom";
+import { gql, useMutation } from "@apollo/client";
+import { useLoginStore } from "../stores/login";
+
+// these schemas will probably change later, all just example data
+const SUBMIT = gql`
+  mutation Login($username: String!, $password: String!) {
+    login(username: $username, password: $password) {
+      username
+      email
+      roles
+    }
+  }
+`;
 
 export const Schedule = () => {
+  const [selectValue, setSelectValue] = useState("");
+  const [submit, { data, loading, error }] = useMutation(SUBMIT);
+
+  const loginState = useLoginStore();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (loginState.loggedIn) {
+      navigate("/");
+    }
+  }, [loginState.loggedIn, navigate]);
+
+  useEffect(() => {
+    if (data && !error && !loading) {
+      // TODO: this route will change to whatever the default page is once the
+      // user is logged in
+      navigate("/");
+    }
+  }, [data, error, loading, navigate]);
+
+  const css = `
+    table {
+      width: 100%;
+      height: 100%;
+    }
+    th {
+      border: 1px solid  #2d3436;
+      font-weight: 300;
+    }
+    .bold{
+      font-weight: bold;
+      border-bottom: 2px solid  #dfe6e9;
+    }
+    .scroll{
+      border: 2px solid  #dfe6e9;
+      overflow-y:auto;
+      max-height:50vh;
+    }
+  `
   return (
-    <></>
+    <Flex
+      w="100%"
+      minH="calc(100vh - 5.5rem)"
+      pt={30}
+      alignItems="center"
+      justifyContent="center"
+      flexDirection="column"
+    >
+      <Container mb={32} maxW="container.xlg">
+        <Heading mb={6}>View Schedule</Heading>
+        <Button
+        mt={5}
+        w="300px"
+        as={Link}
+        to="/schedule"
+        
+        backgroundColor="#8e44ad"
+        colorScheme="purple"
+        textColor="#ffffff"
+        variant="solid"
+        float={`right`}
+      >
+        Generate New Schedule
+      </Button>
+        <Select
+          id="select"
+          w="160px"
+          value={selectValue}
+          mb={5}
+          onChange={(e) => setSelectValue(e.target.value)}
+        >
+          <option value="table">Table View</option>
+          <option value="calendar">Calendar View</option>
+        </Select>
+        <Flex
+          bg="#212938"
+          p={10}
+          borderRadius={10}
+          flexDir="column"
+          style={{ boxShadow: "0px 0px 30px rgba(0, 0, 0, 0.40)" }}
+        >
+         <div className={`scroll`}>
+            <style>
+                {css}
+            </style>
```

Comment:
You can replace all of this with Chakra table components so you don't have to use `className`s and manual CSS

Reply:
Added chakra table, looks different now. Not sure if we like the stripped varient style but i thought it looks nice.

![image](https://user-images.githubusercontent.com/43220800/171930197-0a322af4-bf4a-4b68-96d7-ca8a05ccf72c.png)